tasks/plugins/aiida-bigdft.yml

```diff
@@ -5,9 +5,10 @@
     version: "{{ aiida_bigdft_version }}"
     virtualenv: "{{ aiida_venv }}"
   register: pip_install
-  notify: reentry scan
 
-- meta: flush_handlers
+- name: reentry scan
```

Comment:
this is fine - as mentioned, one other possibility would be to add the reentry scan here:
https://github.com/marvel-nccr/ansible-role-aiida/blob/06e44d91bf14c8c6d60c33987ab61f834380f103/tasks/plugins/code-setup.yml#L2

in which case you need only one

Reply:
Yeh as mentioned, despite the duplication, I think its better logically to run as a consequence of the install, rather than here which is not necessarily triggered